tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -412,3 +414,50 @@ def test_arxiv_update_is_not_store_on_legacy_and_labs(
     assert obj.extra_data['record_matches']
     assert obj.extra_data['skipped-robot-upload']
     assert obj.extra_data['skipped-store-record']
+
+
+def test_article_workflow_error_raises_when_record_is_not_valid(workflow_app):
+    invalid_record = {
+        "titles": [
+            {"title": "A title"}
+        ],
+        "document_type": [
+            "article"
+        ],
+    }
+
+    obj = workflow_object_class.create(
+        data=invalid_record,
+        id_user=1,
+        data_type='hep'
+    )
+    obj_id = obj.id
+
+    with pytest.raises(WorkflowsError):
+        start('article', invalid_record, obj_id)
+
+    obj = workflow_object_class.get(obj_id)
+
+    assert obj.status == ObjectStatus.ERROR
+    assert obj.extra_data['invalid-record'] is True
+    assert obj.extra_data['_error_message'] == 'Workflow record is not schema compliant.'
```

Comment:
Expanding @michamos' comment above, this is where I'd expect to see the validation error message, not something that repeats the same thing that we know from the flag `invalid-record`...

Reply:
Problem is I noticed that the ValidationError message tends to be huge sometimes, but I will think up something.